plugin/hub/hub.go

```diff
@@ -71,14 +71,21 @@ type Hub struct {
 }
 
 func NewHub(skipRegistry bool) *Hub {
-	dir, err := os.Getwd()
-	if err != nil {
-		dir = "."
+
+	var pluginDir string
+	var err error
+	pluginDir = os.Getenv("CQ_PLUGIN_DIR")
```

Comment:
Instead of using os.Getenv i'd suggest to use viper

Reply:
You can add the following to cmd/root.go

rootCmd.PersistentFlags().String( "pluginDir", ".", "Directory to save and load Cloudquery plugins from (env: CQ_PLUGINDIR)")
_ = viper.BindPFlag("plugindir", rootCmd.PersistentFlags().Lookup("pluginDir"))

Then you can change dir, err := os.Getwd() to just dir = viper.GetString("pluginDir")